models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_curve_pools.sql

```diff
@@ -144,25 +149,25 @@ select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 d
 , token_exchange_events as(
     select date_trunc('day', evt_block_time) as time
         , contract_address as pool
-        , sum(case when sold_id = 0 then tokens_sold else (-1) * tokens_bought end) as eth_amount_raw
-        , sum(case when sold_id = 0 then (-1) * tokens_bought else tokens_sold end) as wsteth_amount_raw
+        , sum(case when sold_id = cast(0 as int256) then cast(tokens_sold as double) else (-1) * cast(tokens_bought as double) end) as eth_amount_raw
```

Comment:
minor, but can you do a quick pass to check for how you cast in each of the following scenarios and apply universally?
- string literals: `UINT256 '0'`
- columns: `CAST(column_name as UINT256)`

this was suggested by our query team, it must provide best performance. this would apply to uint256/int256/varbinary/timestamp/date/etc.

a good rule of thumb to do throughout dunesql

Reply:
thanks, fixed